docs/src/theme/DocItem/Metadata/index.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { PageMetadata } from '@docusaurus/theme-common';
+import { useDoc } from '@docusaurus/theme-common/internal';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+export default function DocItemMetadata() {
+  const { metadata, frontMatter, assets } = useDoc();
+
+  if (!metadata.title) {
+    return null;
+  }
+
+  const image = metadata.title
```

Comment:
```suggestion
  const ogImageName = metadata.title
```
a bit more descriptive name. `image` suggests an object that holds the image but here it's just the name